### PR TITLE
20w14a Attributes

### PR DESCRIPTION
**Pull request by liach**
Signed-off-by: liach <liach@users.noreply.github.com>

### CODE REVIEW

File: mappings/net/minecraft/Bootstrap.mapping [hunk 1]
```diff
@@ -11,4 +11,4 @@ CLASS net/minecraft/class_2966 net/minecraft/Bootstrap
 	METHOD method_12851 initialize ()V
 	METHOD method_12852 setOutputStreams ()V
 	METHOD method_17597 getMissingTranslations ()Ljava/util/Set;
-	METHOD method_17598 logMissingTranslations ()V
+	METHOD method_17598 logMissing ()V
```
Comment (YanisBft) [inline mappings/net/minecraft/Bootstrap.mapping:14]: Not only translations I guess?

**Review by YanisBft — APPROVED** [review]

File: mappings/net/minecraft/Bootstrap.mapping (hunk 1 above)
Comment (liach) [reply mappings/net/minecraft/Bootstrap.mapping:14]: yes, now it calls [`DefaultAttributeRegistry.checkMissing()`](https://github.com/FabricMC/yarn/pull/1256/files#diff-8355421ab6feca78702e0ae01b0db30aR4)

**Review by Runemoro — APPROVED** [review]

**Comment by liach** [comment]
@Juuxel For your concern, static methods in a subclass can be named the same as static methods in its superclass, it's just that static methods in a class cannot have same descriptor and name as an instance method (including inherited ones)

**Comment by Juuxel** [comment]
Oh, they're static. That's fine then.

**Pull request merged**